mac

```diff
@@ -234,14 +234,7 @@ elif [ ! -f "$HOME/.ssh/github_rsa.pub" ]; then
   open ~/Applications/GitHub\ Desktop.app
 fi
 
-# git secrets --install -f "$HOME/.git-secrets"
-# git config --global init.templatedir "$HOME/.git-secrets"
-# git config --global core.hooksPath "$HOME/.git-secrets/hooks"
-# git secrets --register-aws --global || true
-# find secret-patterns -type f -name '*.txt' -exec awk 'NF FNR==1{print ""}1' {} + > "$HOME/.git-secrets/patterns"
-# git secrets --add-provider --global -- cat "$HOME/.git-secrets/patterns" || true
-# git secrets --add --allowed --global 'github.com.*/[A-Za-z0-9]{40}' || true
-# git secrets --add --allowed --global 'sha.*[A-Za-z0-9]{40}' || true
-# git secrets --add --allowed --global 'secure:.*' || true
+# TODO: Change to master for merge
+curl -s https://raw.githubusercontent.com/18F/laptop/seekret/seekrets-install | sh -
```

Comment:
Just leaving a reminder here to change the path to point to master before merging.